### PR TITLE
Onboard observability cypress tests

### LINKED ISSUE #74: dashboards-observability
See the main tracking issue for instructions https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/72

### PR DESCRIPTION
**Pull request by davidcui1225**
### Description

Onboard cypress tests for Observability

Note: Currently there are some tests failing that the team is debugging.

### Issues Resolved

#74 

### Check List

- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by tianleh** [comment]
Failed test cases https://github.com/opensearch-project/opensearch-dashboards-functional-test/runs/5456910756?check_suite_focus=true

**Comment by davidcui1225** [comment]
> Failed test cases https://github.com/opensearch-project/opensearch-dashboards-functional-test/runs/5456910756?check_suite_focus=true

These were expected, I am debugging these with the component team and will update the PR tomorrow

File: cypress/integration/plugins/observability-dashboards/event_analytics.spec.js [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
+import { 
+  delayTime,
+  TEST_QUERIES,
+  TESTING_PANEL,
+  SAVE_QUERY1,
```
Comment (Tengda-He) [inline cypress/integration/plugins/observability-dashboards/event_analytics.spec.js:11]: can we name this constant more meaningful ?

File: cypress/utils/plugins/observability-dashboards/otel-v1-apm-service-map-mappings.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (Tengda-He) [inline cypress/utils/plugins/observability-dashboards/otel-v1-apm-service-map-mappings.json:1]: can we create a separate folder for these json files, instead share the same directory as the source code ?

Comment (davidcui1225) [reply cypress/utils/plugins/observability-dashboards/otel-v1-apm-service-map-mappings.json:1]: Sure- I don't think there is something in the onboarding so I will create a `/utils/plugins/<plugin>/json/` directory

**Review by tianleh — APPROVED** [review]

**Review by Tengda-He — APPROVED** [review]

**Pull request merged**